portfolio/src/main/webapp/project_loader.js

```diff
@@ -0,0 +1,70 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// The projects stored as an array of objects, later turned into html
+const projects = [
+  {
+    title: "DIY Digital Watch",
```

Comment:
nit: Most JS users have standardized around using single quotes instead of double quotes. It's an arbitrary decision, it's just easier to be consistent. That lets us search through code without having to account for two different types of quotes.

Reply:
Good to know! Fixed